Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tree-sitter-graph"
-version = "0.11.1"
+version = "0.11.2"
```

Comment:
It might be a borderline case, but I consider this enough to cause a minor bump to 0.12

Reply:
(per slack, I only feel mildly strongly about this, fine with a patch bump too)